File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/shortest_path_length.java
--- 
+++ 
@@ -25,8 +25,7 @@
                 unvisitedNodes.put(nextnode, Integer.MAX_VALUE);
             }
 
-            unvisitedNodes.put(nextnode, Math.min(unvisitedNodes.get(nextnode),
-                    unvisitedNodes.get(nextnode) + length_by_edge.get(Arrays.asList(node, nextnode))));
+            unvisitedNodes.put(nextnode, Math.min(unvisitedNodes.get(nextnode), distance + length_by_edge.get(Arrays.asList(node, nextnode))));
         }
     }
 

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-j/shortest_path_length.py
--- 
+++ 
@@ -17,7 +17,7 @@
             insert_or_update(unvisited_nodes,
                 (min(
                     get(unvisited_nodes, nextnode) or float('inf'),
-                    get(unvisited_nodes, nextnode) + length_by_edge[node, nextnode]
+                    distance + length_by_edge[node, nextnode]
                 ),
                 nextnode)
             )
